src/main/kotlin/no/nav/henvendelsesarkiv/Database.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.henvendelsesarkiv
+
+import no.nav.henvendelsesarkiv.model.Arkivpost
+import no.nav.henvendelsesarkiv.model.ArkivpostMapper
+import no.nav.henvendelsesarkiv.model.Vedlegg
+import no.nav.henvendelsesarkiv.model.VedleggMapper
+import org.springframework.jdbc.core.JdbcTemplate
+
+private const val ARKIVPOST_SQL = """
+            INSERT INTO arkivpost(arkivpostId, arkivertDato, mottattDato, utgaarDato, temagruppe, arkivpostType, dokumentType,
+                                  kryssreferanseId, kanal, aktoerId, fodselsnummer, navIdent, innhold, journalfoerendeEnhet,
+                                  status, kategorikode, signert, erOrganInternt, begrensetPartInnsyn, sensitiv
+            ) VALUES (
+                ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?
+            )
+            """
+
+private const val VEDLEGG_SQL = """
+            INSERT INTO vedlegg(
+                arkivpostId, filnavn, filtype, variantformat, tittel, brevkode, strukturert, dokument
+            ) VALUES (
+                ?, ?, ?, ?, ?, ?, ?, ?
+            )
+            """
+
+class Database constructor(private val jt: JdbcTemplate, private val useHsql: Boolean = false) {
+
+    fun hentHenvendelse(id: Long): Arkivpost? {
+        val arkivpostSql = "SELECT * FROM arkivpost WHERE arkivpostId = ?"
+        val vedleggSql = "SELECT * FROM vedlegg WHERE arkivpostId = ?"
+        val arkivpost = jt.queryForObject(arkivpostSql, ArkivpostMapper(), id)
+        if(arkivpost != null) {
```

Comment:
`pirk-mode: activated`

Mangler mellomrom mellom `if` og if-statementen. Har du sette noe på om det finnes noe linte-verktøy for kotlin? Kanskje det kan legges inn med CodeClimate?

![](https://media.giphy.com/media/AvxviAq9lVCuY/giphy.gif)

Reply:
Skal se om det finnes, det var for såvidt en god ide, så lenge IntelliJ kan plukke den opp :)